### PR TITLE
Ingawei/cashout page

### PR DESCRIPTION
**Pull request by ingawei**
Moved cashout conditions from modal to page, but got rid of this on the cashout page. Want to make sure this won't interrupt anything:
```
  useEffect(() => {
    if (!user) return
    console.log('user', getVerificationStatus(user))
    if (getVerificationStatus(user).status !== 'success') {
      router.push('/gidx/register?redirect=cashout')
    }
  }, [user, router])
  ```
  
Here is what the pages show conditionally.
  
If the getVerificationStatus message is 'User must register' or 'User must verify phone' it shows you this page. @IanPhilips I remember you saying you got rid of the phone verify message or something?
Screenshot 2024-09-06 at 1 43 57 PM (2)

If the getVerificationStatus message is 'User location is blocked', it returns this:
Screenshot 2024-09-06 at 1 46 18 PM

If the getVerificationStatus message is 'User identification failed' or 'User is blocked'
  Screenshot 2024-09-06 at 1 47 06 PM

If the getVerificationStatus failed but is none of the above messages it returns this:
Screenshot 2024-09-06 at 1 47 41 PM

If getVerificationStatus is a success but you have no redeemable sweepscash it returns this:
Screenshot 2024-09-06 at 1 48 52 PM

Finally if it's a success and you have redeemable sweepscash it returns this:
Screenshot 2024-09-06 at 1 49 37 PM

@mantikoros pls lmk if you have any feedback as well

### CODE REVIEW

**Review by mantikoros — APPROVED** [review]
LGTM

- Not sure about kyc redirection flow. Need to see what Ian says.
- Minor issues with the copy. All messages should reference sweepstakes participation, not just cashouts. "Register to participate in sweepstakes games and earn real cash prizes". I'll make a pass through the copy myself.

File: web/pages/cashout.tsx [hunk 1]
```diff
@@ -137,6 +147,102 @@ const CashoutPage = () => {
...
+              >
+                Register and get{' '}
+                <CoinNumber
+                  amount={KYC_VERIFICATION_BONUS}
+                  className={'font-bold'}
+                  isInline
+                />
+              </Link>
+            </Col>
+          ) : message == LOCATION_BLOCKED_MESSAGE ? (
```
Comment (IanPhilips) [inline web/pages/cashout.tsx:180]: `LOCATION_BLOCKED_MESSAGE` is not returned anywhere. You need the private user to find out if their location is blocked, which would be a good thing to show! For that you have to use `locationBlocked`. While you're at it you could use `ageBlocked` and `identityBlocked` for even more informative messaging.

File: web/pages/cashout.tsx [hunk 2]
```diff
@@ -137,6 +147,102 @@ const CashoutPage = () => {
...
+            <Col className="items-center gap-2">
+              <LocationBlockedIcon height={40} className="fill-ink-700" />
+              <div className="text-2xl">Your location is blocked</div>
+              <p className="text-ink-700 text-sm">
+                You are unable to cash out at the moment.
+              </p>
+            </Col>
+          ) : message == IDENTIFICATION_FAILED_MESSAGE ||
```
Comment (IanPhilips) [inline web/pages/cashout.tsx:188]: If they get the `IDENTIFICATION_FAILED_MESSAGE` then you could direct them to the registration flow to try to remedy that state, (could do so with document upload, for example)

**Review by IanPhilips — COMMENTED** [review]
Looks nice! Added a couple notes

**Pull request merged**